cfgov/jinja2/v1/_includes/molecules/pagination.html

```diff
@@ -47,7 +47,7 @@
             <span class="btn_icon__left
                          cf-icon
                          cf-icon-left"></span>
-            Newer
+            {{ prev }}
```

Comment:
Would you consider renaming these `prev_text` and `next_text` just so it's clear in the future what they're supposed to represent?